### PR TITLE
HDDS-2037. Fix hadoop version in pom.ozone.xml.

### PR DESCRIPTION
**Pull request by nandakumar131**

### CODE REVIEW

File: hadoop-hdds/server-scm/pom.xml [hunk 1]
```diff
@@ -100,10 +100,6 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcprov-jdk15on</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.dropwizard.metrics</groupId>
-      <artifactId>metrics-core</artifactId>
```
Comment (adoroszlai) [inline hadoop-hdds/server-scm/pom.xml]: Thanks @nandakumar131 for fixing these Maven warnings, too.

**Comment by nandakumar131** [comment]
@anuengineer Until now all the testing is done with hadoop 3.2.0 and also the previous ozone release (0.4.0-alpha) was done with hadoop 3.2.0 as dependency.
Is there any reason to go to 3.1.0 now?

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 72 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 261 | Maven dependency ordering for branch |
| +1 | mvninstall | 1771 | trunk passed |
| +1 | compile | 1417 | trunk passed |
| +1 | mvnsite | 1706 | trunk passed |
| +1 | shadedclient | 5936 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 517 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 29 | Maven dependency ordering for patch |
| +1 | mvninstall | 2166 | the patch passed |
| +1 | compile | 1418 | the patch passed |
| +1 | javac | 1418 | the patch passed |
| +1 | mvnsite | 1620 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 6 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 777 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 526 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 3594 | root in the patch failed. |
| +1 | asflicense | 60 | The patch does not generate ASF License warnings. |
| | | 16897 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.container.server.TestSecureContainerServer |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClientForAclAuditLog |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.security.TestRaceWhenRelogin |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1351/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1351 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient xml |
| uname | Linux 1f29a6d56193 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 4379370 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1351/2/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1351/2/testReport/ |
| Max. process+thread count | 4893 (vs. ulimit of 5500) |
| modules | C: . hadoop-hdds hadoop-hdds/server-scm hadoop-ozone/ozone-recon U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1351/2/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by elek** [comment]
I think we already started to use 3.2 specific APIs. Some of our requirements are included in hadoop 3.2 but not in 3.1.

If we need any fixes in the hadoop common we should wait until hadoop 3.3. Using 3.1 it would be even harder. 

I would use 3.2 as before.

File: pom.ozone.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
   <properties>
     <!-- required as child projects with different version can't use ${project.version} -->
-    <hadoop.version>3.3.0-SNAPSHOT</hadoop.version>
+    <hadoop.version>3.2.0</hadoop.version>
```
Comment (elek) [inline pom.ozone.xml]: this line can be even deleted AFAIK. There is no project which has this pom.ozone.xml as the parent. Therefore this hadoop.version property is not used at all. (but not  a problem to keep it).

**Pull request closed without merging**